slack/Module.md

```diff
@@ -0,0 +1,67 @@
+## Overview
+This module allows you to access the Slack Web API through Ballerina. This module provides the capability
+to query information from and perform some actions in a Slack workspace.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+
+* Create a Slack account.
+    1. Visit https://slack.com/get-started#/createnew and create a Slack account.
+* Obtain Slack User Oauth Token.
+    1. Visit https://api.slack.com/apps and create a Slack App.
```

Comment:
step 1 is repeated. Fix numbering.

Reply:
Is there any document for instruction on how to obtain Slack User OAuth token?

Reply:
OAuth 2.0

Reply:
> step 1 is repeated. Fix numbering.

Nope. Creating a Slack **Account** and Slack **App** are 2 different things. :)

Reply:
> Is there any document for instruction on how to obtain Slack User OAuth token?

No. There is no documentation provided by Slack. Thats why we have to give the steps here.

Reply:
> OAuth 2.0

This OAuth is not the `OAuth 2.0`. This is a name give to their token by Slack.
Please refer the following screenshot which shows the name of the token.

<img width="520" alt="sss" src="https://user-images.githubusercontent.com/32201965/128465403-ca460f9c-963c-4a61-bc6e-777199a2ce2b.png">